style.css/thirdsectionStyle.css

```diff
@@ -0,0 +1,113 @@
+.third-section {
+  text-align: center;
+}
+
+.title-third-section {
+  background-color: red;
+  color: #ffffff;
+  display: inline-block;
+  padding: 8px;
+  font-weight: bold;
+  margin-bottom: 64px;
+  margin-top: 40px;
+}
+
+.project-list {
+  padding: 0;
+  gap: 25px;
+  display: grid;
+  box-sizing: border-box;
+}
+
+.project-list-item {
+  flex: 50%;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+}
+
+.title-feature {
+  text-align: start;
+  color: red;
+  margin-top: 24px;
+  font-weight: bold;
+}
+
+.content-feature {
```

Comment:
This is bold as well.

Reply:
My bad. It's not bold please remove it. from content-feature class.

Reply:
Done.